Src/BootCamp.Chapter/Player.cs

```diff
@@ -28,34 +34,55 @@ public class Player
         /// <summary>
         /// Player items. There can be multiple of items with same name.
         /// </summary>
-        private Inventory _inventory;
+        private PlayerInventory _inventory;
```

Comment:
Why have a distinction between just inventory? Why is player inventory is any special? Keep it simple, don't be redundant.